lib/linter-views.coffee

```diff
@@ -2,7 +2,6 @@
 
 BottomPanel = require('./views/bottom-panel')
 BottomContainer = require('./views/bottom-container')
-BottomStatus = require('./views/bottom-status')
```

Comment:
Yep, it's an import that I forgot to remove in the recent `Compact Message Storage` PR